features/mbedtls/platform/src/platform_alt.c

```diff
@@ -20,38 +20,33 @@
 
 #include "mbedtls/platform.h"
 #if defined(MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT)
-#include "sns_silib.h"
+mbedtls_platform_context ctx = {0};
 
-/* once https://github.com/ARMmbed/mbedtls/issues/1200 will be supported,
- * rndState should be part of mbedtls_platform_context
- * Until then, we should keep it global and extern */
-
-CRYS_RND_State_t     rndState = { { 0 } } ;
-CRYS_RND_WorkBuff_t  rndWorkBuff = { { 0 } } ;
-
-
-int mbedtls_platform_setup( mbedtls_platform_context *ctx )
+int mbedtls_platform_setup( mbedtls_platform_context *obsolete_ctx )
```

Comment:
It's obsolete, because it's not used and ignored anymore. I can rename

Reply:
If it were obsolete it would no longer be used in any context - but that's not true. We're just choosing not to use it in Mbed OS. It *could* be used in other OS's and platforms.

I really think it's misleading still. Thanks for changing it.